fast_api_als/services/enrich_lead.py

```diff
@@ -12,4 +12,6 @@
 
 
 def get_enriched_lead_json(adf_json: dict) -> dict:
+    if adf_json == {}:
+        logging.warning("Empty JSON")
     pass
```

Comment:
raise KeyError for missing 'adf' key and check if adj_json is valid to raise ValueError